src/proto/grpc/testing/messages.proto

```diff
@@ -38,9 +38,6 @@ package grpc.testing;
 enum PayloadType {
   // Compressable text format.
   COMPRESSABLE = 0;
-
-  // Uncompressable binary format.
-  UNCOMPRESSABLE = 1;
```

Comment:
Given it's still easy to fix whatever breaks, I'd favor removal (ditto for RANDOM, which is already gone in this diff but not on master).